payments-core/src/main/java/com/stripe/android/ConfirmStripeIntentParamsFactory.kt

```diff
@@ -70,30 +73,33 @@ internal class ConfirmPaymentIntentParamsFactory(
     override fun create(
         paymentMethodId: String,
         paymentMethodType: PaymentMethod.Type?,
+        optionsParams: PaymentMethodOptionsParams?,
         requiresSaveOnConfirmation: Boolean,
         recollectedCvc: String?
     ): ConfirmPaymentIntentParams {
+        val fallbackOptions = when (paymentMethodType) {
+            PaymentMethod.Type.Card -> {
+                PaymentMethodOptionsParams.Card(
+                    cvc = recollectedCvc,
```

Comment:
We should be able to do this the right way! We should probably follow up this change with it.

Reply:
Yeah I can do a follow up to add the CVC logic

Reply:
_might_ make more sense to do it here. I'll let you know!